app/src/main/java/com/taskruler/MainViewModel.kt

```diff
@@ -24,7 +24,7 @@ class MainViewModel(var taskService : ITaskService = TaskService()) : ViewModel(
      */
     fun getTasks() {
         viewModelScope.launch {
-            var innerTasks = taskService.getTasks()
+            val innerTasks = taskService.getTasks()
```

Comment:
Because the variable is never modified 'var' is unnecessary, 'val' is sufficient for declaring this variable